scripts/e2e/e2e-dockolith.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+
+LOCAL_DIR=$(cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd)
+
+export CI=true
+export SKIP_SETUP_BUILD=1
```

Comment:
Don't the E2E tests require the module to be built?

Reply:
A build is ran after dockolith is installed, this is just skipping a redundant build